projects/models.py

```diff
@@ -131,10 +131,14 @@ class Project(ModelBase):
         ],
         default=1
     )
+
+    BILLABLE = 0
+    NON_BILLABLE = 1
+
     billable = models.IntegerField(
         help_text='Whether or not the project is chargeable to a'
         ' non-18F client.',
-        choices=[(0, 'Billable'), (1, 'Non-billable')],
+        choices=[(BILLABLE, 'Billable'), (NON_BILLABLE, 'Non-billable')],
```

Comment:
Ok, I guess I'll wait until #38 (which fixes #37) is resolved before continuing with this PR.